bin/list-bin-paths

```diff
@@ -26,6 +26,18 @@ function main() {
 		bin_paths+=("platform-tools")
 	fi
 
+	if [[ -d "${install_path}/emulator" ]]; then
+		bin_paths+=("emulator")
+	fi
+
+	if [[ -d "${install_path}/tools" ]]; then
+		bin_paths+=("tools")
+	fi
+
+	if [[ -d "${install_path}/tools/bin" ]]; then
+		bin_paths+=("tools/bin")
+	fi
```

Comment:
The Android SDK package `tools` is obsolete and was most probably just `cmdline-tools` wrongly published under a wrong name (and has therefore only been published once and been marked as obsolete).  
Additionally this would conflict with the binaries from the `cmdline-tools` package installed by this plugin.  

Therefore I don't see a point adding those to the bin paths.

Reply:
Yes, you are right.
So I commit --amend again for deleting the below code, thanks.

```
if [[ -d "${install_path}/tools" ]]; then
		bin_paths+=("tools")
	fi

	if [[ -d "${install_path}/tools/bin" ]]; then
		bin_paths+=("tools/bin")
	fi
```